lib/common/fse.h

```diff
@@ -137,10 +137,12 @@ FSE_PUBLIC_API unsigned FSE_optimalTableLog(unsigned maxTableLog, size_t srcSize
 /*! FSE_normalizeCount():
     normalize counts so that sum(count[]) == Power_of_2 (2^tableLog)
     'normalizedCounter' is a table of short, of minimum size (maxSymbolValue+1).
+    useLowProbCount is a bool param which is set to 1 to use count=-1 or set to 0 to
```

Comment:
While doc is technically correct, it will be hard for external `fse` users to understand what is going on here.
Also, it's a breaking API change, and there is no other way to request `normalizeCount()`, so a user _must_ understand `useLowProbCount` to invoke this function.

Suggestion : 
- Provide benefits pros/cons of each setting (partially done here for `useLowProbCount==0`). Insist more on compressed / speed benefit, rather than the "technicality" of `count-=1` which might feel obscure at this level of the interface.
- Provide guidance on "good" defaults